lava/proc/lif/models.py

```diff
@@ -7,29 +7,116 @@
 from lava.magma.core.model.py.ports import PyInPort, PyOutPort
 from lava.magma.core.model.py.type import LavaPyType
 from lava.magma.core.resources import CPU
-from lava.magma.core.decorator import implements, requires
+from lava.magma.core.decorator import implements, requires, tag
 from lava.magma.core.model.py.model import PyLoihiProcessModel
 from lava.proc.lif.process import LIF
 
 
 @implements(proc=LIF, protocol=LoihiProtocol)
 @requires(CPU)
-class PyLifModel(PyLoihiProcessModel):
+@tag('floating_pt')
```

Comment:
That's exactly the point: Restrictions as in "well defined boundaries" what you can do rather than anything. It's not that you just tag something. There always must be something else on the other side that deals with the tag.